.circleci/main.yml

```diff
@@ -1260,11 +1260,6 @@ jobs:
             PR=$(gh pr create --base ${parentBranch} --title "bump pach version for release ${CIRCLE_TAG:1}" --body "bump pach version" --reviewer pachydermbuildbot --repo pachyderm/haberdashery)
             sleep 10 # give it a bit time for checks to attach
             gh pr checks ${PR} --watch --interval 10 --repo pachyderm/haberdashery
-            PR_STATE=$(gh pr status --json mergeStateStatus --jq .currentBranch.mergeStateStatus --repo pachyderm/haberdashery)
```

Comment:
I think this might return clean when `config` has passed on circle CI but before the rest of the circle jobs are created. GitHub can't know that there are more jobs coming so it sees the one done job and briefly thinks it's OK. IDK if that's a real problem here, but something to keep in mind.